sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPlugin.scala

```diff
@@ -171,28 +188,44 @@ object ScalaJSPlugin extends AutoPlugin {
 
   import autoImport._
 
-  /** Logs the current statistics about the global IR cache. */
-  def logIRCacheStats(logger: Logger): Unit = {
-    import ScalaJSPluginInternal.globalIRCache
-    logger.debug("Global IR cache stats: " + globalIRCache.stats.logLine)
-  }
-
   override def globalSettings: Seq[Setting[_]] = {
     Seq(
         scalaJSStage := Stage.FastOpt,
 
         scalaJSLinkerConfig := StandardConfig(),
 
+        scalaJSLinkerImpl := {
+          val s = streams.value
+          val log = s.log
+          val retrieveDir = s.cacheDirectory / "scalajs-linker" / scalaJSVersion
+          val lm = {
+            import sbt.librarymanagement.ivy._
+            val ivyConfig = InlineIvyConfiguration().withLog(log)
+            IvyDependencyResolution(ivyConfig)
+          }
```

Comment:
It seems this does not try an load things from local Ivy:

```
[warn] 	module not found: org.scala-js#scalajs-linker_2.12;1.0.0-SNAPSHOT
[warn] ==== public: tried
[warn]   https://repo1.maven.org/maven2/org/scala-js/scalajs-linker_2.12/1.0.0-SNAPSHOT/scalajs-linker_2.12-1.0.0-SNAPSHOT.pom
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: org.scala-js#scalajs-linker_2.12;1.0.0-SNAPSHOT: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
```

@sjrd any idea how to fix that?

Reply:
Er, no, I have no idea :s Perhaps on Gitter: https://gitter.im/sbt/sbt ?

Reply:
Looking at the docs, perhaps all you need is explicitly adding [`Resolver.defaultLocal`](https://www.scala-sbt.org/1.2.8/api/sbt/librarymanagement/Resolver$.html#defaultLocal:sbt.librarymanagement.FileRepository) to the `InlineIvyConfiguration` using its [`withOtherResolvers`](https://www.scala-sbt.org/1.2.8/api/sbt/librarymanagement/ivy/InlineIvyConfiguration.html#withOtherResolvers(otherResolvers:Vector[sbt.librarymanagement.Resolver]):sbt.librarymanagement.ivy.InlineIvyConfiguration) method?